WalletWasabi/Tor/Http/TorHttpClient.cs

```diff
@@ -171,24 +169,19 @@ public async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, Can
 			request.Version = HttpProtocol.HTTP11.Version;
 			request.Headers.AcceptEncoding.Add(new StringWithQualityHeaderValue("gzip"));
 
-			if (TorSocks5Client is null)
+			if (TorTcpConnection is null)
 			{
 				try
 				{
-					TorSocks5Client = new TorSocks5Client(TorSocks5EndPoint!);
-					await TorSocks5Client.ConnectAsync(token).ConfigureAwait(false);
-					await TorSocks5Client.HandshakeAsync(IsolateStream, token).ConfigureAwait(false);
-					await TorSocks5Client.ConnectToDestinationAsync(host, request.RequestUri.Port, token).ConfigureAwait(false);
+					bool useSsl = request.RequestUri.Scheme == "https";
+					int port = request.RequestUri.Port;
 
-					if (request.RequestUri.Scheme == "https")
-					{
-						await TorSocks5Client.UpgradeToSslAsync(host).ConfigureAwait(false);
-					}
```

Comment:
explanation: This part was moved to `TcpConnectionFactory.EstablishConnectionAsync()` function.